### ISSUE TITLE
Multi-line strings in flow mode are prefixed with | (pipe)

### ISSUE THREAD

**Issue by ashald** [go-yaml/yaml#379]
Consider following code:
```go
package main

import (
	"testing"
	"gopkg.in/yaml.v2"
	"log"
)

type T struct {
	Field string `yaml:",flow"`
}

const expected = "{field:foo\nbar\nbaz}"

func TestYamlMultiline(t *testing.T) {
	data := T{Field:"foo\nbar\nbaz"}

	serialized, err := yaml.Marshal(&data)
	if err != nil {
		log.Fatalf("error: %v", err)
	}

	serializedStr := string(serialized[:])

	if serializedStr != expected {
		t.Errorf("Expected '%v', got '%v'", expected, serializedStr)
	}
}

```

Test fails with:
```bash
$ go test -v
=== RUN   TestYamlMultiline
--- FAIL: TestYamlMultiline (0.00s)
	main_test.go:26: Expected '{field:foo
		bar
		baz}', got 'field: |-
		  foo
		  bar
		  baz
		'
FAIL
exit status 1
FAIL	github.com/ashald/yt	0.009s
```

**Comment by nfisher** [comment]
It would be nice to have this behaviour controlled by a field tag. There are a number of ways in which multi-line scalars can be represented in YAML. With CircleCI config it isn't uncommon to use the pipe for multiple lines of a command like this:

```
          command: |
            apt update
            apt install -y wget curl python3-pip
```

https://yaml.org/spec/1.2/spec.html#id2795688

**Comment by exu** [comment]
Is there any update on this ?

**Comment by niemeyer** [comment]
That's fixed in v3 with 513397e.
